src/components/logo/logo.component.tsx

```diff
@@ -1,10 +1,26 @@
+import { ConfigObject } from "@openmrs/esm-config";
 import * as React from "react";
 
-const Logo = ({ width = "110", height = "40" }) => {
+interface LogoProps {
+  config: ConfigObject;
+  width?: number;
+  height?: number;
+}
+
+const Logo: React.FC<LogoProps> = ({ width = 110, height = 40, config }) => {
```

Comment:
No need to pass `config` through as a prop. Eliminate it from the parent component and the props of this component and just do

```typescript
const { logo } = useConfig()
```

in this component.

Reply:
Then to test, mock `useConfig`.

Reply:
Also what's the point of having the width and height as props if they're not passed through by any parent component? This component doesn't need any props at all. Just inline the width and height where they're used. Keep it simple.

Reply:
made the above changes